task/asciidoctor/0.1/README.md

```diff
@@ -0,0 +1,95 @@
+# Asciidoctor
+
+Tekton Task for Asciidoctor https://asciidoctor.org
+
+Asciidoctor is a fast, open source, Ruby-based text processor for parsing AsciiDoc into a document model and converting it to output formats such as HTML 5, DocBook 5, manual pages, PDF, EPUB 3, and other formats.
+
+The resulting document can be sent by email, uploaded to the artifact repository, etc. through other tasks.
+
+## Install the Task
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/asciidoctor/0.1/asciidoctor.yaml
+```
+
+## Pre-requisite
+Install git-clone task from catalog
+```
+https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.3/git-clone.yaml
+```
+
+## Workspaces
+* **source** : A Workspace containing your source directory.
+
+## Parameters
+* **ASCIIDOC_CMD** : Command to convert AsciiDoc files asciidoctor, asciidoctor-pdf, asciidoctor-epub3, etc. Default ```asciidoctor-pdf```.
```

Comment:
can you please list all the possible commands or add a link where a user can find that?

Reply:
Done